### PR TITLE
Quote block: use light block wrapper.

### PR DESCRIPTION
**Pull request by ZebulanStanphill**
## Description
This PR updates the Quote block to use a light block wrapper.

## Checklist:
- [ ] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/quote/edit.js [hunk 1]
```diff
@@ -6,14 +6,17 @@ import classnames from 'classnames';
...
+	__experimentalBlock as Block,
 	BlockControls,
 	RichText,
 } from '@wordpress/block-editor';
-import { BlockQuotation } from '@wordpress/components';
 import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { BlockQuotation } from '@wordpress/primitives';
+
+const BlockWrapper = Block[ BlockQuotation ];
```
Comment (ZebulanStanphill) [inline packages/block-library/src/quote/edit.js:19]: This probably doesn't work on native-mobile. It's worth noting that `BlockQuotation` returns a string ("blockquote") on web and a `View` component on React Native. I have no idea how to get this working with a light block wrapper. Any ideas?

**Comment by ZebulanStanphill** [comment]
Closing in favor of #22803. :smile:

**Comment by ellatrix** [comment]
Oh, sorry. I missed this PR.

**Comment by ZebulanStanphill** [comment]
@ellatrix It's fine! I'm very thankful you picked up the work; I was completely stuck!

**Pull request closed without merging**